packages/clerk-js/src/ui/components/OrganizationProfile/OrganizationGeneralPage.tsx

```diff
@@ -160,7 +160,6 @@ const OrganizationDomainsSection = () => {
                 localizationKey={localizationKeys('organizationProfile.profilePage.domainSection.subtitle')}
                 sx={t => ({
                   paddingLeft: t.space.$9,
-                  marginTop: `-${t.space.$1}`,
```

Comment:
@octoper why did we have this in the first place? 🤔

Reply:
Just to match the gap on the designs but that introduced the problem when focusing, we had removed it from other similar buttons but we forgot this one.